mini_pupper_control/mini_pupper_control/StanfordQuadruped/pupper/HardwareConfig.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 Stanford Student Robotics
```

Comment:
It is defined in mini_pupper_bsp or mini_pupper_2_bsp, so I think this file is not necessary.